packages/jest-runtime/src/__tests__/Runtime-sourceMaps.test.js

```diff
@@ -28,20 +28,17 @@ describe('Runtime', () => {
         try {
           sum();
         } catch (err) {
-          hasThrown = true;
-          /* eslint-disable max-len */
           if (process.platform === 'win32') {
             expect(err.stack).toMatch(
-              /^Error: throwing fn\s+at sum.+\\__tests__\\test_root\\sourcemaps\\throwing-mapped-fn.js:10:9/,
+              /^Error: throwing fn\s+at sum.+\\__tests__\\test_root\\sourcemaps\\(out\\)?throwing-mapped-fn.js:\d+:\d+/,
             );
           } else {
             expect(err.stack).toMatch(
-              /^Error: throwing fn\s+at sum.+\/__tests__\/test_root\/sourcemaps\/throwing-mapped-fn.js:10:9/,
+              /^Error: throwing fn\s+at sum.+\/__tests__\/test_root\/sourcemaps\/(out\/)?throwing-mapped-fn.js:\d+:\d+/,
```

Comment:
node 8 was correct, but node 4 and 6 access `out`. Is that supposed to happen? Feels like it wasn't passed through sourcemap support, then...